### ISSUE TITLE
bootdrive support for stk500v2

### ISSUE THREAD

**Issue by ramarro123** [osbock/Baldwisdom#1]
Hello,

i am trying to use bootdrive on arduino mega, but it fail. i suspect that's because arduino mega use 500v2 proto instead of 500v1?

any plan to update it?

**Comment by osbock** [comment]
Nope. I don't really have a use, but you can do what I did, and adopt the
avrdude code
